static/app/components/button.tsx

```diff
@@ -58,6 +65,15 @@ interface BaseButtonProps
    * See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download
    */
   download?: HTMLAnchorElement['download'];
+  /**
+   * Used when you want to overwrite the default Reload event key for analytics
+   */
+  eventKey?: string;
```

Comment:
Lets name it `analyticsEventKey`

Reply:
Should also be typed as `EventKey`, right?

Reply:
i think in this case i think it is expecting the eventKey to be a string